### DISCUSSION TITLE
Hello World!

### DISCUSSION THREAD

**Discussion by yadvr** [apache/cloudstack#8343] [General]
Hello community, started a test discussion to check the integration between Github and users@ mailing list. Feel free to use for any test/playground, and discuss how to better use Github Discussions for our CloudStack community.

**Comment by weizhouapache** [comment]
Great !

Should this be sent to users@cloudstack.apache.org mailing list as well ? I did not receive an email

**Comment by yadvr** [comment]
Testing - if this message ends up on the users mailing list.

**Reply by DaanHoogland** [reply]
does it?

**Comment by borisstoyanov** [comment]
is there a way to make it two-way integrated with users@, for example not only a copy is being sent to users@, but if someone replies there we get the response here? 
Certainly this is much better experience than emails, having the two-way integration would address all concerns people had during discussing and voting.

**Reply by DaanHoogland** [reply]
voting should not be on users, but on dev@ @borisstoyanov . also I do not think we should do voting here, anything discussed here can be taken to the list, can it?

**Reply by borisstoyanov** [reply]
I don't think you understand me correct Daan, there's no mention of voting here. What I've said is about the concerns raised in discussion and vote threads earlier on mailing lists.

**Reply by DaanHoogland** [reply]
sorry, last word in your comment was "voting"

**Comment by borisstoyanov** [comment]
Also we could probably pin a locked discussion on top in the 'discussions' tab where we describe what and where to find all discussions and this is a secondary option for discussing at the moment?

**Reply by yadvr** [reply]
Yes, looks like we can pin upto 4-5 discussions on the https://github.com/apache/cloudstack/discussions

**Comment by weizhouapache** [comment]
Hi Bobby,

I thought replying to emails directly should work. same as github pull
request/issues.
I have no idea if the integration with the @users mailing list is feasible
or not. Not every email address is linked to a github account.

ps: this reply is sent via email.


-Wei

**Comment by weizhouapache** [comment]
Hi Bobby,

I thought replying to emails directly should work. same as github pull
request/issues.
I have no idea if the integration with the @users mailing list is feasible
or not. Not every email address is linked to a github account.

ps: this reply is sent via email.


-Wei

**Comment by yadvr** [comment]
Closing this :)
